docs/api-reference/composite-layer.md

```diff
@@ -78,6 +78,30 @@ new GeoJsonLayer({
 });
 ```
 
+Example: use customized `ColumnLayer` with a additional updateTrigger (required for the added customization)
```

Comment:
This example may be confusing because it's missing a lot of context. Can you just add an overriding `updateTrigger` e.g. `getIcon` to the old example?

Reply:
Yes, incremental example is good idea, updated.